### ISSUE TITLE
Text Label Needed for Close Button in Flash Message

### ISSUE THREAD

**Issue by joncameron** [avalonmediasystem/avalon#6776]
## Overview
Button missing a text alternative. 
WCAG A 4.1.2: Name, Role, Value

> Buttons should always have a text label. The label can be inserted into the HTML — it does not need to be visible to pass this check.

This can be seen on avalon-dev.

## Done Looks Like
- [ ] Button .btn-close in the flash message body has a text label

**Comment by joncameron** [comment]
👍
